projectforum/projects/project_filters.py

```diff
@@ -5,40 +5,105 @@
+                                                project_list=project_list)
+            return projects_JSON_response(projects)
+        except:
+            return errorMessage(error='Error when sorting by payment')
+    elif order == 'timestamp':
+        try:
+            projects = sort_projects_by_timestamp(ascending=ascending,
+                                                  project_list=project_list)
+            return projects_JSON_response(projects)
+        except:
+            return errorMessage(error='Error when sorting by timestamp')
+    elif order == 'title':
+        try:
+            projects = sort_projects_by_title(ascending=ascending,
+                                              project_list=project_list)
+            return projects_JSON_response(projects)
+        except:
+            return errorMessage(error='Error when sorting by title')
+
+    return errorMessage(error='Something went wrong when handling your query')
+
+
+def get_projects(status, keywords, order, salary, ascending):
+    project_list = filter_projects(status, keywords)
+
+    if len(project_list) == 0:
+        return project_list
+
     if order == 'payment':
-        if salary == 'Hourly':
-            order = '-' + order
-        amount = 'amount'
-        if not ascending:
-            amount = '-' + amount
-        list_of_projects = project_list.order_by(order, amount)
-        quant_proj = check_length(starting_from, ending_at, list_of_projects)
-        if quant_proj == -1:
-            return errorMessage(error="We cannot provide starting from minimum"
-                                      " limit of projects")
-        return projects_JSON_response(list_of_projects[
-                                          starting_from-1:quant_proj
-                                      ])
-    # For every other type of sorting
+        projects = sort_projects_by_payment(salary=salary,
+                                            ascending=ascending,
+                                            project_list=project_list)
+        return projects
+    elif order == 'timestamp':
+        projects = sort_projects_by_timestamp(ascending=ascending,
+                                              project_list=project_list)
+        return projects
+    elif order == 'title':
+        projects = sort_projects_by_title(ascending=ascending,
+                                          project_list=project_list)
+        return projects
+
+
+def sort_projects_by_timestamp(ascending, project_list):
+    order = 'timestamp'
+    if ascending:
+        order = '-' + order
+    list_of_projects = project_list.order_by(order)
+    return list_of_projects
+
+
+def sort_projects_by_title(ascending, project_list):
+    order = 'title'
     if not ascending:
-        # If I'm sorting by descending, add the negative to the query to denote
-        # it
         order = '-' + order
     list_of_projects = project_list.order_by(order)
```

Comment:
How does this order by title? 
A, B, C is fairly easy to understand, but what about AA? Would that go between A and B (as we would naturally think) or after C? What about when the title is multiple words long? "A Great Website" should come before "Awesome Website". Can you add a couple tests for these cases so we can see what is happening in those cases?